README.md

```diff
@@ -22,36 +36,248 @@ component written in native code. Using that component from DNX
+    * Define the environment variable GCLOUD_PROJECT to be your desired project ID. For example:
+
+      ```
+      set GCLOUD_PROJECT=PROJECT_ID
+      ```
+    * If running locally for development/testing, set the project ID using the [Google Cloud SDK][google-cloud-sdk].  Download the SDK if you haven't already, and set the project ID by running the following in the command line:
+
+      ```
+      gcloud config set project PROJECT_ID
+      ```
+
+## Authentication
+Every API call needs to be authenticated. In other to successfully make a call, first ensure that the necessary Google Cloud APIs are enabled for your project and that you've downladed the right set of keys (if it applies to you) as explained in the [authentication document][cloud-common-authentication].
+
+Next, choose a method for authenticating API requests from within your project:
+
+1. When using `gcloud-dotnet` libraries from within Compute/App Engine, no additional authentication steps are necessary.
+2. When using `gcloud-dotnet` libraries elsewhere, you can do one of the following:
+    * Define the environment variable GOOGLE_APPLICATION_CREDENTIALS to be the location of the key.  For example:
+
+      ```
+      set GOOGLE_APPLICATION_CREDENTIALS=/path/to/my/key.json
+      ``` 
+    * If running locally for development/testing, you can authenticate using the [Google Cloud SDK][google-cloud-sdk].  Download the SDK if you haven't already, then login by running the following in the command line:
+
+      ```
+      gcloud auth login
+      ```
+
+## Google BigQuery
+
+- [API Reference][cloud-bigquery-ref]
+- [API Overview][cloud-bigquery-docs]
+
+The BigQuery client library is in pre-release state and is more likely to have significant surface changes over time. Still, if you would like to experiment with it, we would welcome your feedback.
+
+#### Installation
+The package is available on Google's [public MyGet feed][google-myget-feed], where you'll also find the right Package Source to add in Visual Studio. Once done, the package can be installed in the following manner:
+
+```sh
+PM> Install-Package Google.Bigquery.V2 -Prerelease
+```
+
+#### Example
+
+```c#
+using Google.Bigquery.V2;
+...
+
+var client = BigqueryClient.Create(projectId);
+
+// Create the dataset if it doesn't exist.
+var dataset = client.GetOrCreateDataset("mydata");
+
+// Create the table if it doesn't exist.
+var table = dataset.GetOrCreateTable("scores", new SchemaBuilder
+{
+    { "player", SchemaFieldType.String },
+    { "gameStarted", SchemaFieldType.Timestamp },
+    { "score", SchemaFieldType.Integer }
+}.Build());
+
+// Insert a single row. There are many other ways of inserting
+// data into a table.
+table.InsertRow(new Dictionary<string, object>
+{
+    { "player", "Bob" },
+    { "score", 85 },
+    { "gameStarted", new DateTime(2000, 1, 14, 10, 30, 0, DateTimeKind.Utc) }
+});
+```
+
+## Google Cloud Datastore
+
+- [API Reference][cloud-datastore-ref]
```

Comment:
Personally I'd put the links in directly for things that aren't going to be referenced multiple times.